### PR TITLE
Add ESL

### PR DESCRIPTION
**Pull request by basert**
Add ESL (https://account.eslgaming.com - https://www.eslgaming.com)

### CODE REVIEW

File: _data/gaming.yml [hunk 1]
```diff
@@ -84,6 +84,14 @@ websites:
       - totp
     doc: https://www.epicgames.com/help/en-US/c74/c112/a3218
 
+  - name: ESL
+    url: https://account.eslgaming.com
```
Comment (Carlgo11) [inline _data/gaming.yml]: The main url should not be a login page. Change it to this instead:
```suggestion
    url: https://eslgaming.com
```

File: _data/gaming.yml [hunk 2]
```diff
@@ -84,6 +84,14 @@ websites:
...
+    img: esl.png
+    tfa:
+      - totp
+      - u2f
+    doc: https://account.eslgaming.com/mfa
```
Comment (Carlgo11) [inline _data/gaming.yml]: Please change the doc url to something that doesn't require the user to sign in.

**Review by Carlgo11 — CHANGES_REQUESTED** [review]
The logo is unreasonably large. Here's a smaller logo:
esl

File: _data/gaming.yml (hunk 2 above)
Comment (basert) [reply _data/gaming.yml]: We currently don't have a public doc url. Shall I remove the doc link?

File: _data/gaming.yml (hunk 1 above)
Comment (basert) [reply _data/gaming.yml]: Updated to www.eslgaming.com

File: _data/gaming.yml (hunk 2 above)
Comment (Carlgo11) [reply _data/gaming.yml]: If no documentation is available could you please add some screenshots of the 2FA setup process and/or login process?
__Remember to blur or crop out any personal information linked to you.__

You can attach the screenshots here in a comment, like this:
screenshot

Comment (basert) [reply _data/gaming.yml]: * Setup MFA:
Screenshot from 2020-09-02 20-40-19
* Login with TOTP
Screenshot from 2020-09-02 20-36-12
* Login with security key (webauthn)
Screenshot from 2020-09-02 20-35-39

Comment (Carlgo11) [reply _data/gaming.yml]: Thanks! 😄

**Review by Carlgo11 — APPROVED** [review]

**Review by phallobst — APPROVED** [review]

**Pull request merged**